pkg/apis/core/validation/validation.go

```diff
@@ -1753,19 +1753,46 @@ func validatePVSecretReference(secretRef *core.SecretReference, fldPath *field.P
 	return allErrs
 }
 
-func ValidateCSIDriverName(driverName string, fldPath *field.Path) field.ErrorList {
+// ValidateCSRDriverNameOption is an option for ValidateCSIDriverName.
```

Comment:
```suggestion
// ValidateCSIDriverNameOption is an option for ValidateCSIDriverName.
```

Reply:
Doh, thanks for catching this!  Typo'd when renaming from validateOptions .  Fixed now.